nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -2036,7 +2043,7 @@ private Description createErrorDescriptionForNullAssignment(
       String replacement =
           "@SuppressWarnings({"
               + Joiner.on(',').join(Iterables.transform(suppressions, s -> '"' + s + '"'))
-              + "})"
+              + "}) "
```

Comment:
Just formatting?

Reply:
Same, added space between `@SuppressWarnings()` and the code. Otherwise you get `@SuppressWarnings()public void foo()`, which compiles, but looks odd.